engine/table/src/main/java/io/deephaven/engine/util/GroovyDeephavenSession.java

```diff
@@ -354,48 +459,74 @@ public static String removeComments(String s) {
      *         package.class.part.part[.*];"
      */
     @VisibleForTesting
-    public static String isValidImportString(Logger log, String importString) {
+    public Optional<GroovyImport> isValidImportString(String importString) {
```

Comment:
rename to validateImport or createImport?

Also, I left the structure of this method intact, but it could be made to have several returns instead of just one at the end. However, that would remove the `INFO` logging that happens whenever a new import is added... I'm not sure why that exists, but it must be important if we always log it?